dashboard/test/ui/features/teacher_tools/teacher_dashboard/manage_students_tab_views_eyes.feature

```diff
@@ -23,3 +23,42 @@ Feature: Using the manage students tab of the teacher dashboard
     And I see no difference for "manage students tab"
 
     And I close my eyes
+
+  Scenario: Teacher bulk updates US state for all section students
+    Given I create a teacher named "Teacher"
+    And I create a new student section and go home
+    And I save the section id from row 0 of the section table
+    And I create a teacher-associated under-13 student in Colorado named "Student 1" for teacher "Teacher" after CAP start
+    And I create a teacher-associated student named "Student 2" for teacher "Teacher"
+    And I sign in as "Teacher"
+    And I navigate to manage students for the section I saved
+
+    # Test the US state Bulk Set modal
+    When I click selector "#uitest-manage-students-table th:contains(State) i"
+    And I click selector ".pop-up-menu-item:contains(Add state for all students)"
```

Comment:
Would it make sense to use the i18n string for this?

Reply:
No, because we test UI functionality, not i18n

Reply:
We should test the generated content directly, so using dynamic i18n methods is not recommended for this purpose